src/types/IDPEntityDescriptor.ts

```diff
@@ -1,14 +1,12 @@
-/**
- * Do not edit this file it is auto-generated by italia-utils / gen-api-models.
- * See https://github.com/teamdigitale/italia-utils
- */
-/* tslint:disable */
-
 import * as t from "io-ts";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
 
-// required attributes
-const IDPEntityDescriptorR = t.interface({
-  cert: t.readonlyArray(t.string, "array of string"),
+export const IDPEntityDescriptor = t.interface({
+  cert: t.refinement(
```

Comment:
`NonEmptyString` is an io-ts type

```ts
cert: NonEmptyString,
```

Reply:
`cert` must be an array of string. In this case an array of `NonEmptyString`. I’ve add a rype check on the length pf the array that must be non empty. Is wrong?

Reply:
ops sorry, that's my fault, I didn't notice you refined `readonlyArray` - you can actually use [createnonemptyarrayfromarray](https://github.com/gcanti/io-ts-types/blob/82bd6bf09270eb299f0d8140b843ed40a645da8c/docs/modules/fp-ts/createNonEmptyArrayFromArray.ts.md#createnonemptyarrayfromarray-function) from `io-ts-types`:

```ts
  certs: createNonEmptyArrayFromArray(NonEmptyString)
```
```

Reply:
fixed on commit e281711